### PR TITLE
Flask-Dance 0.6.0

### PR DESCRIPTION
**Pull request by singingwolfboy**

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -42,9 +42,9 @@ OAuth authentication for JIRA requires a RSA keypair. To set this up:
 
     .. code-block:: bash
 
-        $ export JIRA_RSA_KEY="$(<jira.pem)"
-        $ export JIRA_CONSUMER_KEY="$(<jira.uuid)"
-        $ heroku config:set JIRA_RSA_KEY=$JIRA_RSA_KEY JIRA_CONSUMER_KEY=$JIRA_CONSUMER_KEY
+        $ export JIRA_OAUTH_RSA_KEY="$(<jira.pem)"
+        $ export JIRA_OAUTH_CONSUMER_KEY="$(<jira.uuid)"
+        $ heroku config:set JIRA_OAUTH_RSA_KEY=$JIRA_OAUTH_RSA_KEY JIRA_OAUTH_CONSUMER_KEY=$JIRA_OAUTH_CONSUMER_KEY
```
Comment (nedbat) [inline docs/install.rst]: Split this line.

**Comment by nedbat** [comment]
:+1:

**Comment by sarina** [comment]
but what does this PR do? there's no description!

**Comment by singingwolfboy** [comment]
Updated Flask-Dance to 0.6.0

**Pull request merged**